modules/cluster/variables.tf

```diff
@@ -60,6 +60,11 @@ variable "ecs_log_level" {
   default     = "info"
 }
 
+variable "retention_in_days" {
+  description = "Log retention given in days."
+  default     = "0"
```

Comment:
Are you sure that 0 is a valid value here? I couldn't find any documentation for it 🤔

Reply:
Well, we have applied 0 in Neo by mistake and it got changed to infinity. I am going to test it again just to be sure.

Reply:
Thanks!